script/manage-account.js

```diff
@@ -40,6 +42,28 @@ changePasswordButton.addEventListener('click', () => {
     changePassword();
 })
 
+newPasswordShow.addEventListener('click', function(event){
+    event.stopPropagation();
```

Comment:
i dont think its needed here,
but you can test it, and remove it when its really not needed